pmm-app/tests/QAN/steps/qanActions.js

```diff
@@ -176,7 +176,7 @@ module.exports = {
     assert.notEqual(
       countAfter,
       countBefore,
-      `After applying Filter value ${countBefore} should not be equal to ${countAfter}`,
+      `After applying a Filter, count of queries should not be equal to ${countAfter}`,
```

Comment:
IMHO, we went from a more explicit to a less explicit error message. Now you have to research what is not equal to `countAfter`.

Reply:
@atymchuk It was done on purpose actually. If the assertion fails it says: After applying Filter value 1 should not be equal to 1

Reply:
lol

Reply:
lol good one, we don't really need to print values, I realized it is already available as part of expected and actual from Assert, we could just make it "After applying a Filter, count of queries was expected to change"

Reply:
then double LOL

Reply:
yeah, we have a lot of such places

Reply:
I'm learning something new every day. Will I ever stop?

Reply:
if you stop learning, I think it would not be a good place to be in :stuck_out_tongue: